Source/Core/BingMapsGeocoderService.js

```diff
@@ -0,0 +1,104 @@
+/*global define*/
+define([
+    './BingMapsApi',
+    './defaultValue',
+    './defineProperties',
+    './loadJsonp',
+    './Rectangle',
+], function(
+    BingMapsApi,
+    defaultValue,
+    defineProperties,
+    loadJsonp,
+    Rectangle) {
+    'use strict';
+
+   var url = 'https://dev.virtualearth.net/REST/v1/Locations';
+
+    /**
+     * Provides geocoding through Bing Maps.
+     * @alias BingMapsGeocoderService
+     *
+     */
+    function BingMapsGeocoderService(options) {
+        options = defaultValue(options, defaultValue.EMPTY_OBJECT);
+        this._canceled = false;
+
+        this._url = 'https://dev.virtualearth.net/REST/v1/Locations';
```

Comment:
I'm not sure it should be possible to supply a different URL, because then it's not really the same geocoder service anymore